src/context/AppProvider/hooks/usePortfolioFetch.tsx

```diff
@@ -1,15 +1,13 @@
 import { useEffect } from 'react'
 
-import { useAccountsFetchQuery } from './useAccountsFetchQuery'
-
 import { useFeatureFlag } from '@/hooks/useFeatureFlag/useFeatureFlag'
 import { useWallet } from '@/hooks/useWallet/useWallet'
 import { portfolioApi } from '@/state/slices/portfolioSlice/portfolioSlice'
 import { selectEnabledWalletAccountIds } from '@/state/slices/selectors'
 import { txHistoryApi } from '@/state/slices/txHistorySlice/txHistorySlice'
 import { useAppDispatch, useAppSelector } from '@/state/store'
 
-export const useAccountsFetch = () => {
+export const usePortfolioFetch = () => {
   const dispatch = useAppDispatch()
   const { isLoadingLocalWallet, modal, wallet } = useWallet().state
   const enabledWalletAccountIds = useAppSelector(selectEnabledWalletAccountIds)
```

Comment:
q: do we want to err on the side of perf and *actually* add discovery checks i.e this guy ?https://github.com/shapeshift/web/pull/9993#discussion_r2214218093 

Conceptually speaking, looks like we wouldn't want to start fetching portfolio until we've discovered accounts?
That piggybacks to the notion of reactivity and "low priority" things

Reply:
Sounds sane to me, indeed v. flaky!
And yeah, that part looks like it could finally get the love it needs re: loading states, so happy to dedicate a follow-up PR to it 🙏🏽  that super ugly loading state atm was mostly for perf reasons, and I believe we now may be able to avoid it (and keep those accounts in an ugly skeleton state, despite them actually being loaded), if we fetch all portfolio at once.